userspace/libscap/engine/gvisor/gvisor.h

```diff
@@ -50,7 +50,6 @@ struct parse_result {
     // pointers to each encoded event within the supplied output buffer
 	std::vector<scap_evt*> scap_events;
 };
-typedef struct parse_result parse_result;
```

Comment:
Probably we still need this since in `parser.cpp` we use the typedef `parse_result`

Reply:
@loresuso 's point here is that C++ does not need `typedef`s, those are only necessary in C.

Reply:
Oh sorry I didn't know that